docs/resources/placement_group.md

```diff
@@ -0,0 +1,51 @@
+---
+page_title: "Linode: linode_placement_group"
+description: |-
+  Manages a Linode Placement Group.
+---
+
+# linode\_placement\_group
+
+Manages a Linode Placement Group.
+
+## Example Usage
+
+Create a Placement Group with the local anti-affinity policy:
+
+```terraform
+resource "linode_placement_group" "test" {
+    label = "my-placement-group"
+    region = "us-mia"
+    affinity_type = "anti_affinity:local"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `label` - (Required) The label of the Placement Group. This field can only contain ASCII letters, digits and dashes.
+
+* `region` - (Required) The region of the Placement Group.
+
+* `affinity_type` - (Required) The affinity policy to use when placing Linodes in this group.
+
+* `is_strict` - (Optional) Whether Linodes must be able to become compliant during assignment. (Default `true`)
```

Comment:
The default in Cloud Manager is true but the field is required by the API. Would it make sense to make the field required in TF too?